client/src/app/queries/facts.ts

```diff
@@ -0,0 +1,34 @@
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+
+import { getFacts } from "@app/api/rest";
+import { AxiosError } from "axios";
+import { Application, Fact } from "@app/api/models";
+import { ApplicationsQueryKey } from "./applications";
+
+export const FactsQueryKey = "facts";
+
+export const useFetchFacts = (applicationID: number | string | undefined) => {
+  const { data, isLoading, error, refetch } = useQuery(
+    [FactsQueryKey, applicationID],
+    {
+      queryFn: async () => await getFacts(applicationID),
+      enabled: !!applicationID,
+      onError: (error: AxiosError) => console.log("error, ", error),
+      select: (facts): Fact[] => {
+        return Object.keys(facts).map((fact) => {
+          return {
+            name: fact,
+            data: facts[fact],
+          };
+        });
+      },
+    }
```

Comment:
There is potential to make this shorter (and look more functional)....

```suggestion
      select: (facts): Fact[] =>
        Object.keys(facts).map(
          (fact) => ({ name: fact, data: facts[fact] })
        ),
```